### ISSUE TITLE
Add Calamine (Pure RUST) support for Google Sheets

### ISSUE THREAD

**Issue by ghost** [tafia/calamine#162]
RUST Crate google_sheets
https://docs.rs/google-sheets4/1.0.12+20190625/google_sheets4/

Akin to Google Sheets Python API
https://github.com/burnash/gspread

**Comment by tafia** [comment]
You mean adding calamine support for google sheet right?
I guess, this could be done behind a feature flag (I'd rather avoid adding all these dependencies in the regular case).

**Comment by Heijns** [comment]
Yes, a feature flag would be a good approach since login and edit history are specific to Google Sheets. I've renamed the title.

**Comment by jmcnamara** [comment]
Closing as not planned.
